js_basics/task5-1.js

```diff
@@ -0,0 +1,5 @@
+let radius = 5.25;
+const pi = 3.14;
+let circleArea = pi * Math.pow(radius, 2);
+console.log(circleArea);
+console.log(Number.parseFloat(circleArea).toFixed(2));
```

Comment:
`Number.parseFloat` call is unnecessary since circleArea is type of number

Reply:
fixed it